src/System.Security.Cryptography.Xml/tests/XmlDsigXsltTransformTest.cs

```diff
@@ -182,8 +182,11 @@ public void OnlyInner()
             doc.LoadXml(test);
 
             transform.LoadInnerXml(doc.ChildNodes);
-            Stream s = (Stream)transform.GetOutput();
-            Assert.Throws<ArgumentNullException>(() => Stream2Array(s));
+            Assert.Throws<ArgumentNullException>(() =>
```

Comment:
What's supposed to ANE here?  If it's checking that transform.GetOutput() is null then assert that it's null...